src/main/kotlin/org/jetbrains/research/libsl/visitors/FunctionVisitor.kt

```diff
@@ -20,6 +17,9 @@ class FunctionVisitor(
     val errorManager: ErrorManager
 ) : LibSLParserVisitor<Unit>(functionContext) {
     private lateinit var buildingFunction: Function
+    private lateinit var buildingConstructor: Constructor
```

Comment:
Yeah, refactoring of constructors/destructors/functions/procedures ([functions.kt](https://github.com/vpa-research/libsl-parser/pull/7/files#diff-0a82144a6ca600f6859ede7c2a71d15975799cfd445b9174bede429679b604c3)) is needed here